plugins/header_rewrite/conditions.h

```diff
@@ -547,7 +547,7 @@ class ConditionStringLiteral : public Condition
   typedef Matchers<std::string> MatcherType;
 
 public:
-  ConditionStringLiteral(const std::string &v);
+  explicit ConditionStringLiteral(const std::string &v);
```

Comment:
I'd recommend `std::string_view` - it's almost always the better choice for passing in a string. For example, if the call is
```
ConditionStringLiteral("Battle Beast");
```
then a temporary `std::string` will be created, copying the string. For `std::string_view` a temporary will also be made but it won't copy the string.

Reply:
Future optimizations.